config-templates/config.php

```diff
@@ -861,6 +861,37 @@
      */
     'production' => true,
 
+    /*
+     * Set this option to adjust basic caching directives sent by module.php.
+     *
+     * Defaults to ['public' => true, 'max_age' => 86400]
+     */
+    // 'module.php.cache' => ['public' => true, 'max_age' => 86400],
```

Comment:
I think the configuration could use some refactor and rewording. Naming is always hard. In this case, I don't think "`module.php.*`" tells anything of what this config options do to someone who's not looking at this PR. Also, our current practice is to try to group together configuration options that are related (e.g. the [`application` option](https://github.com/simplesamlphp/simplesamlphp/blob/master/config-templates/config.php#L36-L52) or the [`debug` option](https://github.com/simplesamlphp/simplesamlphp/blob/master/config-templates/config.php#L220-L256)). This makes it easier to configure the software, in my opinion, and to document options that are related. I would follow the same approach.

As I said, naming is hard, but since this is related to caching of assets, would it make sense to call it just `assets`, then add a `caching` option inside? In my head, this makes sense as we might want to configure other stuff about assets in the future (e.g. the base URL for them, if we want to offload assets to their own server). In this case, and as far as I can see, almost all the available options that can be passed to symfony are set in the `Cache-Control` HTTP header, and there are directives for `ETag` and `Last-Modified`. I think it makes sense to have something like this then:

```php
    /**
     * Explanation of the assets directive
     */
    'assets' => [
        'caching' => [
           /**
            * Explanation of the supported caching-related directives
            */
            'public' => true,
            'max_age' => 86400,
            'etag' => false,
            'last_modified' => ...
        ],
    ],
```